airflow/www/utils.py

```diff
@@ -219,7 +219,16 @@ def get_params(**kwargs):
     return urlencode({d: v for d, v in kwargs.items() if v is not None}, True)
 
 
-def generate_pages(current_page, num_of_pages, search=None, status=None, tags=None, window=7):
+def generate_pages(
+    current_page,
+    num_of_pages,
+    search=None,
+    status=None,
+    tags=None,
+    window=7,
+    sorting_key=None,
+    sorting_direction=None,
```

Comment:
You need to update the docstring as well.

Reply:
Thanks. Adding it.